lib/commands/diff/index.js

```diff
@@ -0,0 +1,25 @@
+var child = require("child_process");
+var pager = require("default-pager");
+var chalk = require("chalk");
+var path  = require("path");
+var fs    = require("fs");
+
+module.exports = function (config, cli) {
+  var packageName = cli.input[1];
+  var filePath = "packages";
+
+  if (packageName) {
+    var pkgLoc = path.join(config.packagesLoc, packageName, "package.json");
+
+    if (!fs.existsSync(pkgLoc)) {
+      console.log(chalk.red("Package '" + packageName + "' does not exist."));
+      process.exit(1);
+    }
+
+    filePath += "/" + packageName;
+  }
+
+  var diff = child.spawn("git", ["diff", "--color", filePath]);
```

Comment:
> Do we want to check against the last tag?

Oh right duh.

> Or is that another cli parameter to pass in?

I think this should just be a simple command to see everything since the last release